.evergreen/config.yml

```diff
@@ -869,6 +889,39 @@ axes:
         variables:
            STORAGE_ENGINE: "inmemory"
 
+task_groups:
```

Comment:
It's important that drivers use task groups or `post` to ensure the serverless instances get reaped properly, even on test failure.